### ISSUE TITLE
Implement the Kernel-VM facade interface

### ISSUE THREAD

**Issue by semuxgo** [semuxproject/semux-core#55]
Depends on #53 

To fully integrate EVM into Semux, the database/core layers need to be refactored to implemented the facade interfaces specified in the `vm.client` package. Thereafter, the VM can read real data from and write data back to the Semux blockchain.

**Comment by semuxgo** [comment]
Or we add a few wrapper classes to implement the facade interfaces

**Comment by orogvany** [comment]
I think I've implemented them pretty OK in facade interfaces.  I think long term we might want to just implement interfaces for most of them inline, rather than facades, but some seem better as facades.. i'm up for either or a mix.

**Comment by orogvany** [comment]
The facade classes are all implemented, can probably close this one

**Comment by orogvany** [comment]
Closing as this has been merged into develop/1.5.0RC1
